backend/tarsy/integrations/llm/client.py

```
@@ -258,14 +314,69 @@
+            details.append(f"RootCause={type(root_cause).__name__}: {str(root_cause)}")
+        
+        # Use vars() to dump all instance variables
+        try:
+            exception_vars = vars(exception)
+            if exception_vars:
+                for key, value in exception_vars.items():
+                    str_value = repr(value)
+                    if len(str_value) > 200:
+                        str_value = str_value[:200] + "..."
+                    details.append(f"{key}={str_value}")
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)